test/jsonSchema.test.ts

```diff
@@ -152,6 +195,48 @@ describe('Json Schema Conversion Test File', function () {
         },
       ],
     },
+    time_field: {
+      value: 1726067527,
+      unit: 'MILLIS',
+      isAdjustedToUTC: true,
+    },
```

Comment:
is it redundant to send all these fields everytime setting time_field for example, if not then this ties to same question about units and how they are defined in schema

Would love some suggestions